api/v1alpha1/clusterfederatedtrustdomain_webhook.go

```diff
@@ -53,7 +53,7 @@ func (r *ClusterFederatedTrustDomain) ValidateCreate() (admission.Warnings, erro
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
-func (r *ClusterFederatedTrustDomain) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
+func (r *ClusterFederatedTrustDomain) ValidateUpdate(_ runtime.Object) (admission.Warnings, error) {
```

Comment:
can you remove `_` when no parametter isued? 
```suggestion
func (r *ClusterFederatedTrustDomain) ValidateUpdate(runtime.Object) (admission.Warnings, error) {
```
same comment for all cases like this

Reply:
done